flink-core/src/main/java/org/apache/flink/api/common/state/StateDescriptor.java

```diff
@@ -258,7 +258,7 @@ public boolean isQueryable() {
      * Configures optional activation of state time-to-live (TTL).
      *
      * <p>State user value will expire, become unavailable and be cleaned up in storage depending on
-     * configured {@link StateTtlConfig}.
+     * configured {@link StateTtlConfig}, and the field defaultValue will be invalid.
```

Comment:
I think the explanation of the change of `defaultValue` could be placed in another statement starts with `<p>`.